cross/x265/Makefile

```diff
@@ -0,0 +1,62 @@
+PKG_NAME = x265
+PKG_VERS = 1.9
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)_$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://bitbucket.org/multicoreware/x265/downloads
+PKG_DIR = $(PKG_NAME)_$(PKG_VERS)/source
+
+DEPENDS =
+
+HOMEPAGE = http://x265.org/
+COMMENT  = x265 is an open source HEVC encoder.
+LICENSE  = GPL
+
+CONFIGURE_TARGET = myConfigure
+
+include ../../mk/spksrc.cross-cc.mk
+
+YASM_PATH = $(WORK_DIR)/../../../native/yasm/work-native/install/usr/local/bin
+
+CMAKE_ARGS = -DCMAKE_INSTALL_PREFIX=$(STAGING_INSTALL_PREFIX)
```

Comment:
Move to above the include for consistency's sake. There are probably multiple locations in the PR where this applies.